backend-api/src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -78,13 +78,7 @@ export async function lambdaHandlerConstructor(
   );
   if (verifyTokenSignatureResult.isError) {
     const errorMessage = verifyTokenSignatureResult.value.errorMessage;
-    if (verifyTokenSignatureResult.value.errorCategory === "SERVER_ERROR") {
```

Comment:
I'm sorry - please may you help me work out why this can't return a server error? Slow brain